src/CoreWCF.BuildTools/src/OperationParameterInjectionGenerator.SourceGenerationSpec.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+
+namespace CoreWCF.BuildTools
+{
```

Comment:
Inconsistent use of file-scoped namespaces. Some new files have braces like this one, and some have no braces like PrivateServiceContractAnalyzer. What are you thoughts on consistency through the project? I like adopting newer cleaner ways of writing code, but I'd really like everything at least within a single project to be consistent. Should we update the entire project (not a massive project so not a big effort)? Or have new files use this older braces style? Or do you think it's not a big deal to be mixed?

Reply:
> Inconsistent use of file-scoped namespaces. Some new files have braces like this one, and some have no braces like PrivateServiceContractAnalyzer. What are you thoughts on consistency through the project? I like adopting newer cleaner ways of writing code, but I'd really like everything at least within a single project to be consistent. Should we update the entire project (not a massive project so not a big effort)? Or have new files use this older braces style? Or do you think it's not a big deal to be mixed?

I think the solution does contain old style c# syntax (I'm thinking to code that has been ported to modern .NET) that would deserve a refresh but can live as is as it's easing comparison to .NetFX or SSM repo. 
However for new code like the one from BuildTools we should be consistent across the project and aligned with today's standards. I will fix that.